.github/workflows/test.yaml

```diff
@@ -0,0 +1,42 @@
+on:
+  push:
+    paths-ignore:
+      - 'docs/**'
+      - 'README.md'
```

Comment:
Biggest reason to have "lint" in a separate file is that we can skip running unit/smoke tests for documentation only changes, but the commit linting still applies.